Project.toml

```diff
@@ -42,6 +44,7 @@ Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
 OceananigansEnzymeExt = "Enzyme"
 
 [compat]
+AMDGPU = "0.6"
```

Comment:
need to add compat entry for GPUArrays also

Reply:
AMDGPU needs to be at least 0.8 otherwise we get a build error related to the use of opaque pointers. Not entirely sure where in the code LLVM chokes, but it occurs when using a rectilinear grid with ROCmGPU architecture

Reply:
compat for GPUArrays added; set to v0.9.1